nautobot/extras/tests/test_relationships.py

```diff
@@ -384,6 +384,46 @@ def setUp(self):
         for cra in self.invalid_relationship_associations:
             cra.validated_save()
 
+    def test_exception_not_raised_when_updating_instance_with_relationship_type_o2o_or_o2m(self):
+        """Validate 'Unable to create more than one relationship-association...' not raise when updating instance with
+        type one-to-one, symmetric-one-to-one, one-to-many relationship."""
+
+        # Assert Exception not raise updating source of RelationshipAssociation with one-to-many relationship type
+        cra_1 = RelationshipAssociation(relationship=self.o2m_1, source=self.sites[0], destination=self.vlans[1])
+        cra_1.validated_save()
```

Comment:
Given the original reported issue, it would be good for at least one of these tests to check calling `validated_save()` a second time *without* making any data changes.